msamp/deepspeed/runtime/zero/fp8_stage_1_and_2.py

```diff
@@ -325,8 +325,9 @@ def independent_gradient_partition_epilogue(self):    # noqa: C901
                 else:
                     avg_new = self.fp8_get_flat_partition(self.fp8_params_in_partition[i])
                     for accumulated_grad, new_avg_grad in zip(self.fp8_averaged_gradients[i], avg_new):
-                        accumulated_grad.data = (accumulated_grad.float() +
-                                                 new_avg_grad.float()).cast(WEIGHT_GRAD_QTYPE, in_time=True)
+                        accumulated_grad.data.copy_(
```

Comment:
Why use copy_? I know _copy could work, but it will cause additional data copy.

Reply:
I have removed the additional data copy..
For this change, the condition of `ValueError('This ScalingTensor is locked.')` in `msamp/common/tensor/tensor.py` has been updated.